Replicator/IncomingRev.cc

```diff
@@ -64,8 +64,9 @@ namespace litecore { namespace repl {
 
     // Read the 'rev' message, then parse either synchronously or asynchronously.
     // This runs on the caller's (Puller's) thread.
-    void IncomingRev::handleRev(blip::MessageIn *msg) {
+    void IncomingRev::handleRev(blip::MessageIn *msg, uint64_t bodySizeOfRemoteSequence) {
```

Comment:
Just `bodySize` is a better name. It does get stored in a map keyed by remote sequence ID, but that's not relevant at this point. It's just the (approximate) size in bytes of the revision body according to the peer.

Reply:
Okay, will change it